Cookbook/Technical-Documents/CRP-Bot.md

```diff
@@ -0,0 +1,99 @@
+# CRP Bot
+
+## What is the CRP process and how does the CRP bot work
+
+Because we're now certified ISO-13485, our SSDLC and the ISO process requires that we track every changes in the codebase and in each release of each flavor of our apps. This means that we need to track which commits (and which corresponding JIRA tickets) have been included in each version of the apps we push to the stores.
+
+In order to comply with that tracking requirement, every time a version of one of the flavors (Babylon UK, Bupa, Telus, Ascension US, ...) is released to one of the stores (AppStore or PlayStore), and the CRP bot is triggered (either using the Slack command – e.g. `/crp ios release/babylon/4.4.0` – or automatically during release cutoff _(soon)_) to handle that part of the release process, the bot will handle the following tasks:
```

Comment:
TODO: remove that `_(soon)_` once the implementation of that automation (merged yesterday) is confirmed to be working (on next release cut tomorrow)
```suggestion
In order to comply with that tracking requirement, every time a version of one of the flavors (Babylon UK, Bupa, Telus, Ascension US, ...) is released to one of the stores (AppStore or PlayStore), and the CRP bot is triggered (either automatically during release cutoff, or when using the Slack command – e.g. `/crp ios release/babylon/4.4.0` to trigger it manually if ever needed) to handle that part of the release process, the bot will handle the following tasks:
```